src/changelog/2023/10/resource-alerts.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Resource Monitoring in Audit Log
+date: 2023-10-26 10:00:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+
+In the FlowFuse Audit Log of an instance, it will now display when the resource utilization of CPU or memory exceeds 75% over a period of five minutes. This should provide transparency regarding resource utilization and serve as an indicator for choosing the appropriate instance size.
```

Comment:
Can we link to: https://flowfuse.com/docs/user/logs/ and update that page?

Reply:
Yes good point!
https://github.com/FlowFuse/flowfuse/pull/3008

Reply:
```suggestion
In the FlowFuse Audit Log (see [Documentation](https://flowfuse.com/docs/user/logs/#audit-log))of an instance, it will now display when the resource utilization of CPU or memory exceeds 75% over a period of five minutes. This should provide transparency regarding resource utilization and serve as an indicator for choosing the appropriate instance size.
```